users/serializers.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any, List
+
+from rest_framework_simplejwt.serializers import TokenObtainPairSerializer
+from rest_framework_simplejwt.state import User
+from rest_framework_simplejwt.tokens import RefreshToken
+
+
+class CustomTokenObtainPairSerializer(TokenObtainPairSerializer):
+
+    def validate(self, attrs: List[str]) -> Any:
```

Comment:
Any will work, but it looks like data is some kind of dictionary

Reply:
I tried that but mypy sees the return type as Any 
![image](https://user-images.githubusercontent.com/58403334/112752010-1ea50480-8fd1-11eb-83f5-de9443ab902e.png)